app/models/user.rb

```diff
@@ -1,24 +1,22 @@
-class User < ApplicationRecord
-  # Associations
-  # A user has many posts
-  # A user has many comments
-  # A user has many likes
-  has_many :posts, foreign_key: :author_id
-  has_many :comments, foreign_key: :author_id
-  has_many :likes, foreign_key: :author_id
-
-  # after_save :update_posts_counter
-
-  # The 3 most recent posts for a user
-  def recent_posts
-    posts.order(created_at: :desc).limit(3)
-  end
-
-  # Updates the posts counter for a user
-  def update_posts_counter
-    update(posts_counter: posts.count)
-  end
-
-  validates :name, presence: true
-  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-end
+class User < ApplicationRecord
+  before_validation :set_default
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  # :Confirmable

```

Comment:
- [ ] As required for the project, kindly ensure that your application asks for confirmation of email i.e `:confirmable` should also be enabled in the user model (user.rb file) and working as expected